lib/Cpdf.php

```diff
@@ -5164,7 +5164,7 @@ public function getTextWidth(float $size, string $text, float $wordSpacing = 0.0
             $this->selectFont($this->defaultFont);
         }
 
-        $text = str_replace(["\r", "\n"], "", $text);
+        $text = preg_replace('/[\x00-\x1F\x7F\xA0]/u', '', $text);
```

Comment:
I think `\xA0` slipped in there accidentally. Would be nice to have a short comment that this is about non-printable characters here.

Reply:
indeed